### PR TITLE
evolution: handling of seed = None for pypet

### PR DESCRIPTION
**Pull request by caglorithm**
`pypet` doesn't like to add `None` results. The RNG seed parameter `model.params["seed"]` is usually `None` and was set to `0` to avoid issues before. Now it stays `None` but when saving to `pypet`, the case is handled.

The rest of the other changes are only black formatting...

### CODE REVIEW

File: neurolib/optimize/evolution/evolution.py [hunk 1]
```diff
@@ -315,22 +341,33 @@ def initPypetTrajectory(self, traj, paramInterval, POP_SIZE, NGEN, model):
...
         # if a model was given, save its parameters
         # NOTE: Convert model.params to dict() since it is a dotdict() and pypet doesn't like that
         if model is not None:
-            traj.f_add_result("params", dict(model.params), comment="Default parameters")
+            params_dict = dict(model.params)
+            # replace all None with zeros, pypet doesn't like None
+            for key, value in params_dict.items():
+                if value is None:
+                    params_dict[key] = 0
+            traj.f_add_result("params", params_dict, comment="Default parameters")
```
Comment (caglorithm) [inline neurolib/optimize/evolution/evolution.py:362]: This is the part that is new

File: neurolib/models/wc/loadDefaultParams.py [hunk 2]
```diff
@@ -23,12 +23,7 @@ def loadDefaultParams(Cmat=None, Dmat=None, seed=None):
     params.dt = 0.1  # ms 0.1ms is reasonable
     params.duration = 2000  # Simulation duration (ms)
     np.random.seed(seed)  # seed for RNG of noise and ICs
-    # set seed to 0 if None, pypet will complain otherwise
-    params.seed = seed or 0
-
-    # make sure that seed=0 remains None
-    if seed == 0:
-        seed = None
```
Comment (caglorithm) [inline neurolib/models/wc/loadDefaultParams.py:31]: and this is what I got rid of in every `loadDefaultParams.py`

File: neurolib/optimize/evolution/evolution.py (hunk 1 above)
Comment (jajcayn) [reply neurolib/optimize/evolution/evolution.py:362]: so None seed is saved to pypet as 0 seed. Am I right? What if you actually use 0 seed, that one is also saved as 0 to pypet?

Comment (caglorithm) [reply neurolib/optimize/evolution/evolution.py:362]: That's still an unresolved issue tbh... Was an issue before but I didn't want to spend too much time with it.

Comment (jajcayn) [reply neurolib/optimize/evolution/evolution.py:362]: what if seed specifically is saves as string? `str(<any_integer>) = '<any_integer>'` and `str(None) = 'None'`.. than on loading you can simply:
```python
try:
    seed = int(<saved_in_pypet>)
except ValueError:
    seed = None

```

Comment (jajcayn) [reply neurolib/optimize/evolution/evolution.py:362]: AFAIK `pypet` can do strings...

Comment (caglorithm) [reply neurolib/optimize/evolution/evolution.py]: it does, nice!

**Pull request merged**